test/integration/targets/groupby_filter/runme.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# We don't set -u here, due to pypa/virtualenv#150
+set -e
```

Comment:
Sorry, my life apparently revolves around food, and I pushed before I was ready so I could eat dinner.  The recent update should restore `-x` and remove some explicit verbosity in my printing.